lib/IO/Socket/Async/SSL.pm6

```diff
@@ -296,11 +301,11 @@ class IO::Socket::Async::SSL {
                    :$enc = 'utf8', :$scheduler = $*SCHEDULER,
                    OpenSSL::ProtocolVersion :$version = -1,
                    :$ca-file, :$ca-path, :$insecure, :$alpn,
-                   Str :$ciphers --> Promise) {
+                   Str :$ciphers, :$sslkeylogfile --> Promise) {
```

Comment:
I think `ssl-key-log-file` would be more fitting with the naming conventions here overall.

Reply:
Done.